File: Iniciante/URI 1930.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var line = lines.shift().split(' ');
var T1 = parseInt(line[0]);
var T2 = parseInt(line[1]);
var T3 = parseInt(line[2]);
var T4 = parseInt(line[3]);

var totalTomadas = (T1 + T2 + T3 + T4) - 3;
console.log(totalTomadas);

File: Iniciante/URI 1930.r
input = file('stdin', 'r')

line = strsplit(readLines(input, n=1), " ")
t1 = as.integer(line[[1]][1])
t2 = as.integer(line[[1]][2])
t3 = as.integer(line[[1]][3])
t4 = as.integer(line[[1]][4])
totalTomadas = (t1 + t2 + t3 + t4) - 3;

write(sprintf("%d", totalTomadas), '')